metta/dependent-types/ModusPonensDTL.metta

```diff
@@ -0,0 +1,22 @@
+;; MeTTa port for modus ponens PLN rule
+;; as DTL (Dependent Type Language)
+;;
+;; (≞ P ptv)
+;; (≞ (→ P Q) pqtv)
+;; ⊢
+;; (≞ Q qtv)
+;;
+;; where ptv and pqtv are truth values of the premises P and (→ P Q) respectively.
+;; qtv represents the resulting truth value of the conclusion.
```

Comment:
Alternatively you could have that comment appear here.  Maybe it's better cause users may not bother looking into the formula.

Reply:
I've just added the comment in this file, and since the letter P is also used in the comment to denote probability so I changed the original usage of P and Q to A and B to avoid any potential confusion to the users, which might seem inconsistent with other files, please let me know if you think it's inappropriate and I can change it back, thanks!